README.md

```diff
@@ -18,8 +18,9 @@
 ## Usage
 
 1. Choose your flavour.
-2. Copy the contents of `themes/<flavour>.cava` into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
-3. Reload cava if it was already playing.
+2. Choose your background variant: opaque or transparent.
+3. Copy the contents of `themes/<flavour>.cava` or `themes/<flavour>-transparent.cava` into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
```

Comment:
```suggestion
3. Copy the contents of `themes/<flavor>.cava` or `themes/<flavor>-transparent.cava` into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
```
Minor spelling nit!

Reply:
Good catch

I've also applied the same fix to point `1.` of the README